src/commands.ts

```diff
@@ -322,6 +323,12 @@ export const inputHandlers = (store: Store<State, any>) => ({
       globals.suppressExpansion = false
     }
 
+    // Handle Alt+ArrowLeft when caret is at the beginning of a thought
```

Comment:
Try to avoid explaining functionality by saying "Handle...". The problem is that it doesn't really explain it. Handle it how? A better comment would be one that explains _why_ this is needed, what would happen if we _didn't_ do it, and any limitations on our knowledge.

```
// For some reason, when the caret is at the beginning of the thought, alt + ArrowLeft sets the caret to the end.
// Prevent this default behavior, as the caret should have nowhere to go when it is already at the beginning.
```